code/game/objects/items/halloweendecorations.dm

```diff
@@ -0,0 +1,37 @@
+//Two small halloween decoration items
+//Candle pile
+/obj/item/flame/waxcandles
+	name = "candle pile"
+	desc = "A pile of half molten white wax candles, that seem to burn on forever."
+	icon = 'icons/obj/contained_items/halloween_decorations.dmi'
+	icon_state = "candles"
+	item_state = "candles"
+	light_color = "#E09D37"
+	var/wax = 40000
```

Comment:
They are a different kind of candle as a decoration. They are also starting lit up, thats why I decided to make a seperate item. I was not sure if these things stay since its for halloween specifically.

Reply:
They also burn substantially longer than regular candles.

Reply:
i think matt means you could make it a child of the candle item instead, to reduce code copy-paste.